qa/deepsea/deepsea.yaml

```diff
@@ -11,3 +11,6 @@ tasks:
         - deepsea.ceph_conf:
         - deepsea.orch:
                 stage: 3
+        - deepsea.create_pools:
+        - deepsea.orch:
+                stage: 4
```

Comment:
I'd rather not add this to `qa/deepsea/deepsea.yaml`, since I'm using it for tests that don't need Stage 4 at all.

Maybe it would make sense to have a `qa/deepsea/stage4.yaml` though.